### PR TITLE
add more takeaways and separate by topic

### PR DESCRIPTION
**Pull request by kweav**
Updated `pages/index.Rmd` to include more takeaways and specifically separate them out by topic (linking the relevant page).

### CODE REVIEW

**Comment by kweav** [comment]
I'm not sure if I'm a fan of separating them out by topic. The original looks more polished.

**Comment by kweav** [comment]
Do we need the flowchart image to be higher resolution?

**Comment by avahoffman** [comment]
I think it looks pretty good! My concern is a really long bulleted list otherwise.

Yes, we'll want the flowchart in high res -- this could just be looking strange on the preview though.

**Comment by kweav** [comment]
I rebuilt the flowchart directly into that slidedeck -- it's a little different spacing wise now (seems bigger/more cramped), but does seem to be higher res on the website. What do you think? I prefer the old one, but want it to be a good resolution, so...

I first to use an svg in the slidedeck, but google slides doesn't seem to support

**Comment by avahoffman** [comment]
Thanks for doing all that! We might need to use a static file. It still looks a bit pixely unfortunately

**Comment by kweav** [comment]
I think the static png looks all right!

File: pages/index.Rmd [hunk 1]
```diff
@@ -24,9 +37,9 @@ Click on each tab to learn more.
...
 This user poll was conducted in Spring 2024, with the poll open for responses from February 15th to March 25th. 52 total responses were received, two of which were determined to be duplicate users, leaving a total of **50 user responses** used in the analysis. The following graphic shows the arrangement of questions. The first question was used to separate users into "Current" and "Potential" users. Demographics, experience, awareness, and preference related questions were asked of all users.
 
-```{r, fig.align='center', fig.alt= "Example image", echo = FALSE, out.width="100%"}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1PANHDY3T9wpEX1GBYcqkdYMG0c20UeZPH2pwjHFcLik/edit#slide=id.g2dda935efac_0_0")
-```
-
+![](../plots/AnVILPoll2024Flowchart.png)
 
 
+```{r, eval=FALSE, fig.align='center', fig.alt= "Example image", echo = FALSE, out.width="100%"}
```
Comment (avahoffman) [inline pages/index.Rmd]: Do we need to keep this chunk?

File: pages/index.Rmd [hunk 2]
```diff
@@ -8,13 +8,26 @@ output:
...
+- [*Identify User Type*](https://hutchdatascience.org/AnVIL_Poll_2024/IdentifyTypeOfUsers.html)
+  -  The majority of current users leverage AnVIL for ongoing projects. Potential users were evenly split between never having used the AnVIL (but have heard of it) and having used AnVIL at some point previously.
+- [*Demographics*](https://hutchdatascience.org/AnVIL_Poll_2024/Demographics.html)
+  - Most of the respondents have a PhD or are currently working on a PhD.
+  - Many respondents do computational work.
+  - Almost all respondents are affiliated with a research intensive institution.
+  - PRIMED, GREGoR, and eMERGE were the most popular consortia affiliations. CCDG and GTEx are also represented.
+- [*Experience*](https://hutchdatascience.org/AnVIL_Poll_2024/Experience.html)
+  - Overall, respondents report less comfort with containers or workflows than various programming languages and their IDEs.
+  - Many respondents reported being extremely experienced with human genomic data. Far fewer respondents reported being extremely experienced with human clinical or non-human genomic data.
```
Comment (avahoffman) [inline pages/index.Rmd]:
```suggestion
  - Many respondents report being extremely experienced with human genomic data. Far fewer respondents report being extremely experienced with human clinical or non-human genomic data.
```
Double check for tense consistency

File: pages/index.Rmd [hunk 3]
```diff
@@ -8,13 +8,26 @@ output:
...
+  - Of the survey provided choices, respondents reported accessing or having interest in accessing All of Us, UK Biobank, and GTEx the most over other controlled access datasets.
+- [*Awareness*](https://hutchdatascience.org/AnVIL_Poll_2024/Awareness.html)
+  - Most respondents are aware of AnVIL Demos, even if they haven't attended one.
+  - Most respondents are aware of the AnVIL Support Forum, even if they haven't used it.
+- [*Preferences*](https://hutchdatascience.org/AnVIL_Poll_2024/Preferences.html)
+  - All respondents rank having specific tools or datasets supported/available as a very important feature for using the AnVIL (and this feature is ranked the highest by *current users*); *Potential users* rate having a free version with limited compute or storage as the most important feature for their potential use of the AnVIL.
+  - All respondents prefer virtual training workshops over other training modalities
```
Comment (avahoffman) [inline pages/index.Rmd]:
```suggestion
  - All respondents prefer virtual training workshops over other training modalities.
```

**Review by avahoffman — APPROVED** [review]
Minor tweaks and I think it's ready!

File: pages/index.Rmd (hunk 1 above)
Comment (kweav) [reply pages/index.Rmd]: I agree we don't need to if you like the static image!

**Comment by kweav** [comment]
@avahoffman did a read through to sync verb tenses more. How's it look? I'll go through takeaways on individual pages too and put that in another branch/PR if changes are needed

**Review by avahoffman — APPROVED** [review]
lgtm!

**Pull request merged**